### PR TITLE
Add Landing Page

### PR DESCRIPTION
**Pull request by nicocadq**
NOTE: new feature for release

### CODE REVIEW

**Comment by LautaroPardo0210** [comment]
Task linked: [CU-20qg9m9 Add landing page](https://app.clickup.com/t/20qg9m9)

File: src/pages/Landing/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+          , para que los voluntarios puedan entenderte mejor a la hora de
+          evaluar tus necesidades.
+        </p>
+      </section>
+      <section
+        className={[styles.banner, styles.gradient, styles['get-started']].join(
+          ' '
+        )}
+      >
+        <h1>¿Que tal te parece nuestra propuesta?</h1>
```
Comment (rubenderigo) [inline src/pages/Landing/LandingPage.js]:
```suggestion
        <h1>¿Qué tal te parece nuestra propuesta?</h1>
```

File: src/pages/Landing/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          <span className={[styles.highlight, styles.secondary].join(' ')}>
+            Solicitudes
+          </span>{' '}
+          donde podrás plantear tus necesidades y estas serán visibles por todos
+          los usuarios, teniendo todos los datos necesarios para contactarse
+          contigo.
+        </p>
+      </section>
+      <section className={[styles.banner, styles.tertiary].join(' ')}>
+        <p>¿Como hago para mejorar mi perfil?</p>
```
Comment (rubenderigo) [inline src/pages/Landing/LandingPage.js]:
```suggestion
        <p>¿Cómo hago para mejorar mi perfil?</p>
```

File: src/pages/Landing/LandingPage.js [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+      <section className={styles.banner}>
+        <p>¿Te gustaría darle un segundo uso a tus cosas?</p>
+      </section>
+      <section className={styles.section}>
+        <div>
+          <img src={IMackDesktopMock} alt="Vista Desktop Web" />
+        </div>
+        <p className={styles.description}>
+          <span className={styles.bold}>
+            ¿Tienes algo para donar pero no sabes como hacerlo?
```
Comment (rubenderigo) [inline src/pages/Landing/LandingPage.js]:
```suggestion
            ¿Tienes algo para donar pero no sabes cómo hacerlo?
```

**Review by rubenderigo — APPROVED** [review]

**Pull request merged**